plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -208,9 +212,7 @@ export class PolicesServer {
 
       const processedPolicies = await this.processPolicies(policyRaw);
 
-      for (const policy of processedPolicies) {
-        await this.enforcer.addOrUpdatePolicy(policy, 'rest', false);
-      }
+      await this.enforcer.addPolicies(processedPolicies, 'rest');
```

Comment:
Same as the comment on `await this.enforcer.addOrUpdateGroupingPolicy(role, 'rest', false);`